test/xmlenc.encryptedkey.js

```diff
@@ -65,15 +62,18 @@ describe('encrypt', function() {
       keyEncryptionAlgorighm: 'http://www.w3.org/2001/04/xmlenc#rsa-oaep-mgf1p'
     };
 
-    crypto.randomBytes(32, function(err, randomBytes) {
+    var plaintext = 'The quick brown fox jumps over the lazy dog';
```

Comment:
I think it's best to test this with a constant value that we can check across versions of Node.